src/components/NavBar.vue

```diff
@@ -0,0 +1,85 @@
+<script>
+import { RouterLink } from "vue-router";
+</script>
+
+<template>
+  <header>
+    <div class="logo-tagline-div">
+      <div class="logo-div">
+        <img require src="../assets/Worth_Wild.png" class="nav-logo" alt="worth wild logo" />
```

Comment:
I think "Worth_Wild" needs to be "worth_wild" for this build error to go away? (lower case 'w')